### PR TITLE
L2 Configurator Diagram

### PR DESCRIPTION
**Pull request by scott-silver**
Quick sketch of what we were discussing earlier:

image

### CODE REVIEW

File: diagrams/l2_configurator_diagram.uml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+@startuml
+
+rectangle "L1" {
+  rectangle GovSimple
```
Comment (kevincheng96) [inline diagrams/l2_configurator_diagram.uml:4]: nit: This will probably just be `Governor`

**Review by kevincheng96 — COMMENTED** [review]
🆒

File: diagrams/l2_configurator_diagram.uml (hunk 1 above)
Comment (scott-silver) [reply diagrams/l2_configurator_diagram.uml]: updated

**Comment by scott-silver** [comment]
closing for now; diagram is incomplete

**Pull request closed without merging**